src/styles/text/canvas_text.js

```diff
@@ -351,8 +351,9 @@ class MultiLine {
         this.height = 0;
         this.lines = [];
 
-        this.ellipsis = '...';
+        this.ellipsis = '…';
```

Comment:
Sorry, I probably shouldn't have suggested this at all. We really should research handling across fonts more in depth before changing. I suspect it may not exist in some custom fonts and could just render with unexpected results?